DC-631 Next Core package release (#49)
madewithkoji/koji-core: README.md

import { KojiBackend } from '@withkoji/core';
```

> Initialize the package with your configuration data, and use `KojiBackend.middleware` to scope operations per customization.

Reviewer: ```suggestion
Initialize the package with your configuration data, and use `KojiBackend.middleware` to scope operations per creator app.
```
Customization is a bit unclear here, but I'm not sure if my suggestion clarifies enough. The idea is that the middleware allows each creator's app to use its own data.

Author: BEFORE: ... to scope operations per customization.
AFTER: ... to scope operations for each customized version of the app.

// Parse application/json
app.use(bodyParser.json());

> // Use Koji's middleware to handle scoping across your app's customizations

Reviewer: ```suggestion
// Use Koji's middleware to handle scoping across customized versions of the app
```
Same as the previous comment. "Customizations" feels right when talking about the options provided to the creator. We need a way to talk about the customized app for each creator.

Author: Fixed.

## Related resources

* [Package documentation](https://developer.withkoji.com/reference/core/withkoji-koji-core)
* [Tutorial: Koji quick start](https://developer.withkoji.com/docs/getting-started/start-tutorial)

> * [Migrating to the Koji core package](https://developer.withkoji.com/docs/getting-started/migrate-koji-core)

Reviewer: ```suggestion
* [Migrating to the Koji core package](https://developer.withkoji.com/docs/dev-guides/migrate-koji-core)
```

Looks like the Gatsby site handles the link redirect, but let's update it to the latest since we're here.

Author: Fixed.
